go-fuzz/hub.go

```diff
@@ -116,6 +216,29 @@ func newHub(metadata MetaData) *Hub {
 			ro.intLits = append(ro.intLits, []byte(lit.Val))
 		}
 	}
+
+	if dictPath != "" {
+		ro.strLits = nil // Discard existing tokens
```

Comment:
This means we still have `ro.intLits`: I am wondering how AFL/libfuzzer acts here with the dict option, if they still use int literals internally or maybe drop them at all. (I bet they actually still use some int literals generators, because that may be useful, heh)

Reply:
As far as I know, that depends on the user, because you have to explicitly run libtokencap to extract the tokens and use them. Do you think we should set `ro.intLits` also to nil? Adding int values can be accomplished in the dictionary by having a token containing the bytes of the int.

Reply:
Honestly I don't know... let's maybe leave it as it is, but let's document it somewhere?

Reply:
My gut tells me that integer literals don't skew the signal:noise ratio of the token list built from scanning source code— I suggest we leave it as-is.